src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSExitConditions.cs

```diff
@@ -151,28 +153,53 @@ public IList<PSExitCodeMapping> ExitCodes
             }
         }
         
-        public PSExitOptions SchedulingError
+        public PSExitOptions FileUploadError
```

Comment:
mapping and obsolete?

Reply:
My intent is to just leave this as is (no obsolete) since this is a breaking change, unless otherwise suggested by one of the powershell maintainers.